sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCoalesceBatches.scala

```diff
@@ -58,7 +59,7 @@ object ConcatAndConsumeAll {
    * @param dataTypes the output types.
    * @return a single batch with all of them concated together.
    */
-  def buildNonEmptyBatchFromTypes(arrayOfBatches: Array[ColumnarBatch],
+  def buildNonEmptyBatchFromTypes(arrayOfBatches: Seq[ColumnarBatch],
```

Comment:
Why Seq and not Array? Also we should change the name of the variable if we are going to change the type.

Reply:
I was using Seq for the collection of SpillableColumnarBatch to split, and changed it because I thought it was a bit more permissive (allowed ArrayBuffer, and Seq). I changed it back to Array, and adjusted types in https://github.com/NVIDIA/spark-rapids/pull/7852/commits/3a5c1d0dbaad4c2dcaf663dcaf4c58beb0d537db